gbpservice/neutron/config/nsf.ini

```diff
@@ -0,0 +1,13 @@
+[DEFAULT]
+debug=True
+rabbit_password=openstack 
+rabbit_userid=stackrabbit
+rabbit_hosts=<openstack controller ip adress:rabbitPort>  #Example 192.168.2.107:5672
+rabbit_port=5672
+rabbit_host=<openstack controller ip address>
```

Comment:
Yes those will be derived from neutron.conf, they were put in separate .ini file to highlight what parameters the framework need. Finally the framework run as /usr/bin/python /usr/bin/nfp --config-file /etc/neutron/neutron.conf --config-file /etc/neutron/nfp.ini